### PR TITLE
GAWB-2939 make library test handle when there are many existing published datasets

### PR DESCRIPTION
**Pull request by andy7i**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [ ] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by davidangb — APPROVED** [review]
:+1: code looks correct; I did not test myself. I assume if tests pass this change is functionally correct!

**Comment by andy7i** [comment]
jenkins retest

**Comment by andy7i** [comment]
Jenkins failure was unrelated

File: README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Support for running FireCloud outside of the Broad is planned.
 
 ### Development Stack Setup
 
-1. Make sure you've run the `firecloud-setup.sh` script in the `firecloud-develop` repo to render the config files and run script.
+1. Make sure you've run the `firecloud-setup.sh` script in the `local` branch of the `firecloud-develop` repo to render the config files and run script.
```
Comment (zarsky-broad) [inline README.md]: The script will be in all branches of the repo once Jackie's PR is merged in.

Comment (andy7i) [reply README.md]: That's good to know. I'll just remove this change then?

**Comment by andy7i** [comment]
Tests passed prior to latest change to README file.

**Comment by andy7i** [comment]
jenkins retest

**Review by gdlex4015 — APPROVED** [review]

**Pull request merged**